.circleci/config.yml

```
@@ -0,0 +1,41 @@
+version: 2.1
+
+x-shared-config:
+  - &yarn_restore_cache
+    keys:
+      - yarn-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - yarn-v1-{{ .Branch }}-
+      - yarn-v1-
+  - &yarn_save_cache
+    key: yarn-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node-docker:
+    docker:
+      - image: mutations/node:16.13.0-alpine3.14-2021-11-04
```

This should be updated to `mutations/node:16.13.2-alpine3.15-2022-01-18` to match the version in the [`docker-compose.yml`](https://github.com/mutations/wait-for-netlify-action/pull/2/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R15) from #2.

```suggestion
      - image: mutations/node:16.13.2-alpine3.15-2022-01-18
```